event_registration_multi_qty/models/event.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel<sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class EventEvent(models.Model):
+    _inherit = 'event.event'
+
+    registration_multi_qty = fields.Boolean(
+        string='Registration Multi Qty',
+        help='Allow more than one registration'
+    )
+
+    @api.onchange('registration_multi_qty')
+    def _onchange_registration_multi_qty(self):
+        if not self.registration_multi_qty:
+            self.registration_ids.write({'qty': 1})
```

Comment:
Calling `write` under an `onchange` is dangerous, because onchanges work in a draft record while `write` don't. You should call `update` instead.

Anyway, this does not seem to be a good choice. If somebody clicks here by mistake, he'd be overriding all qtys. I feel for this is better to add a `constraint`, so that if you already have registrations with `qty != 1` you cannot check this box. The same constraint could be called when changing `qty` field in registrations.

----

Further reading showed me that [after all, `qty` is ignored if disabled](https://github.com/OCA/event/pull/78/files#diff-e670b96f19d2dfc8404e228ec4a39b0aR27), so why even bother about this?